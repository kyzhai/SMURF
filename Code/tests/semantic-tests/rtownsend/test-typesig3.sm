/* Basic polymorphic type */


a = 3
a :: b

main = []
