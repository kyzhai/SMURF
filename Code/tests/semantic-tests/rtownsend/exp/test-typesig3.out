Fatal error: exception Sast.Type_mismatch("a")
